security/pkg/nodeagent/sds/monitoring.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sds
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	totalPushCounts = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "citadel_agent",
+		Subsystem: "sds_service",
+		Name:      "total_push_count",
```

Comment:
Probably should be prefixed with citadel? Or is that what the namespace/subsystem does? If so that is pretty cool

Reply:
FWIW, pilot's is named `pilot_xds_pushes` with label `type=(eds|lds|cds|rds)`. Ideally we would probably have just a general `istio_xds_pushes` and make this `type=sds`, but probably can't do that for backwards compat reasons